phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixEmbeddedDriver.java

```diff
@@ -349,7 +349,7 @@ public ConnectionInfo normalize(ReadOnlyProps props, Properties info) throws SQL
             }
             if(principal == null){
                 if (!isConnectionless) {
-                   principal = props.get(QueryServices.HBASE_CLIENT_PRINCIPAL);
+                    principal = props.get(QueryServices.HBASE_CLIENT_PRINCIPAL);
```

Comment:
Unnecessary?

Reply:
In some of the earlier commits there were changes in ths area. This line fixes the identation.